wpiutil/src/main/native/include/wpi/deprecated.h

```diff
@@ -5,10 +5,6 @@
 #ifndef WPIUTIL_WPI_DEPRECATED_H_
 #define WPIUTIL_WPI_DEPRECATED_H_
 
-#ifndef WPI_DEPRECATED
-#define WPI_DEPRECATED(msg) [[deprecated(msg)]]
-#endif
```

Comment:
I would probably leave this here for a bit; vendors may well be using it.

Reply:
Time to deprecate our deprecation macro. :stuck_out_tongue: